src/tests/test_verify_schema.sh

```diff
@@ -1,23 +1,44 @@
 #!/bin/bash -e
 
-PYTHONPATH="$PYTHONPATH:$TRAVIS_BUILD_DIR/src"
+# Use the Travis build dir if specified, otherwise assume that the
+# repo is unpacked in the current working dir
+REPO_ROOT_DIR=${TRAVIS_BUILD_DIR:-.}
+
+PYTHONPATH="$PYTHONPATH:$REPO_ROOT_DIR/src"
 export PYTHONPATH
 
 function verify_xml {
     # Verifies XML data against the schema
     xml="$1"
     type="$2"
     echo "Validating $type XML schema..."
-    xmllint --noout --schema "$TRAVIS_BUILD_DIR/src/schema/$type.xsd" $xml
+    xmllint --noout --schema "$REPO_ROOT_DIR/src/schema/$type.xsd" $xml
 }
 
-if [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then
+if [[ $TRAVIS_PULL_REQUEST == "false" ||
+      $GH_EVENT == 'push' &&
+      $GITHUB_REPOSITORY == 'opensciencegrid/topology' ]]; then
+    # Ensure that the .ssh dir exists
+    mkdir ~/.ssh
+    chmod 0700 ~/.ssh
     cat >> ~/.ssh/known_hosts <<EOF
```

Comment:
not sure what the default umask is in this context - if it's `0022` you should be fine, but if it's `0002` i think you need to ensure `~/.ssh/known_hosts` is not group-writable.

Reply:
Good call. Fixed.